bin/add-roles.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+set -eo pipefail
```

Comment:
Nice and easy but...
... has it been tested in an actual failure condition?

What will be the resulting output of this in case of error?

Can I ask that you put a screenshot evidence of what will happen as a result of this change when an error occurs?

Reply:
Nevermind. It's not going to be possible in preview.

Hopefully it will work.

Conversation resolved.